packages/verified-fetch/test/verified-fetch.spec.ts

```diff
@@ -210,6 +239,34 @@ describe('@helia/verifed-fetch', () => {
       expect(resp.statusText).to.equal('Not Implemented')
     })
 
+    it('can round trip json via .json()', async () => {
+      const obj = {
+        hello: 'world'
+      }
+      const j = json(helia)
+      const cid = await j.add(obj)
+
+      const resp = await verifiedFetch.fetch(cid)
+      expect(resp.headers.get('content-type')).to.equal('application/json')
+
+      const output = await resp.json()
+      await expect(j.add(output)).to.eventually.deep.equal(cid)
+    })
+
+    it('can round trip json via .arrayBuffer()', async () => {
+      const obj = {
+        hello: 'world'
+      }
+      const j = json(helia)
+      const cid = await j.add(obj)
+
+      const resp = await verifiedFetch.fetch(cid)
+      expect(resp.headers.get('content-type')).to.equal('application/json')
+
+      const output = ipldJson.decode(new Uint8Array(await resp.arrayBuffer()))
```

Comment:
After https://github.com/multiformats/js-multiformats/pull/287 we can drop the `Uint8Array` wrapper:

```suggestion
      const output = ipldJson.decode(await resp.arrayBuffer())
```

Reply:
This has been done now